cola/widgets/startup.py

```diff
@@ -7,18 +7,29 @@
 from qtpy import QtWidgets
 
 from ..i18n import N_
+from .. import cmds
 from .. import core
 from .. import display
 from .. import guicmds
 from .. import hotkeys
 from .. import icons
 from .. import qtutils
+from .. import utils
 from .. import version
 from . import clone
 from . import defs
 from . import standard
 
 
+ICON_MODE = 0
+LIST_MODE = 1
+
+LIST_VIEW_BOOKMARK = 0
+LIST_VIEW_RECENT = 1
+
+LABEL_TITLE_FONT_SIZE = 16
```

Comment:
We probably don't want to hard-code a font size... I'll see where it's used to suggest an alternative.

Reply:
Yes, as this comment, I'll improve it.

>Instead of using a hard-coded font size, lookup the current font size and then scale by some factor (eg. 0.8). Is the default size too small or too large?